doc/source/contributor.rst

```diff
@@ -0,0 +1,222 @@
+.. _developerdocumentation:
+
+=======================
+Developer Documentation
+=======================
+
+In this section, you will find documentation relevant to developing
+socok8s.
+
+.. _contributing:
+
+Contributor Guidelines
+======================
+
+Submodules
+----------
+
+This repository uses submodules. All of the following guidelines apply
+to this socok8s repository only. Please check other projects' practices
+before contributing to them, if a change needs to happen there.
+
+Before submitting code
+----------------------
+
+This is a very fast moving project, and you should probably contact us
+before starting to work on it.
+
+If you're willing to submit code, please remember the following rules:
+
+* All code should match our
+  :ref:`codeguidelines`.
+* All code requires to go through our :ref:`reviews`.
+* Documentation should be provided with the
+  code directly. See also :ref:`documentation`.
+
+.. _bug_reporting:
+
+Bug reporting process
+---------------------
+
+Bugs should be filed as Github issues.
+
+When submitting a bug, or working on a bug, please ensure the following
+criteria are met:
+
+* The description clearly states or describes the original problem or root
+  cause of the problem.
+* The description clearly states the expected outcome of the user action.
+* Include historical information on how the problem was identified.
+* Any relevant logs or user configuration are included, either directly
+  or through a pastebin.
+* If the issue is a bug that needs fixing in a branch other than master,
+  please note the associated branch within the launchpad issue.
+* The provided information should be totally self-contained. External access
+  to web services/sites should not be needed.
+* Steps to reproduce the problem if possible.
+
+.. _reviews:
+
+Review process
+--------------
+
+Any new code will be reviewed before merging into our repositories.
+
+2 votes are required before merging a patch.
```

Comment:
We don't really have the concept of +2 (vs +1) in github. Do you mean approval from committers?

Reply:
Yeah 2 approving reviews.

Reply:
ah, of course :-p